job/src/run_jump_model.py

```diff
@@ -81,6 +81,8 @@ def run_jump_model(run_uuid):
     name = 'run_jump_model'
     data = get_input_dict_from_run_uuid(run_uuid)
     user_uuid = data.get('user_uuid')
+    
+    data.pop('user_uuid',None) # Remove user uuid from inputs dict to avoid downstream errors
```

Comment:
@rathod-b was this something you discussed with Nick? Are there any downsides to doing this or potential issues?